deepchem/models/torch_models/ESM2.py

```diff
@@ -0,0 +1,96 @@
+from typing import Dict, Any
+from deepchem.models.torch_models.hf_models import HuggingFaceModel
+from transformers import AutoTokenizer, AutoConfig, AutoModelForSequenceClassification
+from transformers.modeling_utils import PreTrainedModel
+
+
+class ESM2(HuggingFaceModel):
```

Comment:
Test cases. You need to add the following tests:

- Basic fitting test
- Basic inference test